packages/block-library/src/image/edit.js

```diff
@@ -432,6 +432,7 @@ class ImageEdit extends Component {
 								<Fragment>
 									{ getInspectorControls( imageWidth, imageHeight ) }
 									<ResizableBox
+										className="wp-block-image__resizer"
```

Comment:
This should be `block-library-image__edit-resizer` if we want to be strict about our className guidelines :)

Reply:
yes, I think so the `wp-block-*` convention is only for styles that can be used on the frontend, which I think is not the case here.